tests/masonry/utils/test_package_include.py

```diff
@@ -65,6 +65,20 @@ def test_pep_561_stub_only_package_good_name_suffix() -> None:
     ]
 
 
+def test_pep_561_stub_only_partial_namespace_package_good_name_suffix() -> None:
+    pkg_include = PackageInclude(
+        base=fixtures_dir / "pep_561_stub_only_partial_namespace", include="good-stubs"
+    )
+    print(pkg_include.elements)
```

Comment:
```suggestion

```

Reply:
Thanks, didn't notice I had left that lying around